packages/manager/pages/properties/[propertyName]/index.tsx

```diff
@@ -73,49 +75,49 @@ const StyledText = styled(Text)`
   cursor: pointer;
 `;
 
+const StyledCodeBlock = styled(CodeBlock)`
+  text-align: left;
+`;
+
 export const getServerSideProps = async (context: ContextProps) => {
     try {
         const token = (await getSession(context as any) as any).accessToken;
         const propertyReq = getPropertyRequest(context);
         const urlEvent = getEventAnalyticsUrl();
+        const spaListUrl = getSpaListUrl(propertyReq);
         const payloadActivites = {
             "activities": {
                 "propertyName": propertyReq
             }
         };
-        const payloadCount = {
-            "count": {
-                "propertyName": propertyReq
-            }
-        };
         const response = await Promise.all(
             [
                 await post<Properties>(urlEvent, payloadActivites, token),
-                await post<Properties>(urlEvent, payloadCount, token)
+                await get<Properties>(spaListUrl, token),
             ]
         );
-        const [activitesResponse, countResponse]: AnyProps = response;
-        if (activitesResponse == null || countResponse == null) {
-            return { props: { url: getGuideUrl() } };
-        }
+        const [activitiesResponse, countResponse]: AnyProps = response;
+        // if (!!activitiesResponse?.length || !!countResponse?.length) {
+        //     return { props: { url: getGuideUrl() } };
+        // }
```

Comment:
Remove commented code!

Reply:
Sorry it got commited, I'm still working on this and it was committed 😅